README.md

```diff
@@ -51,6 +51,7 @@ Flags:
   --default_priority=5         Annotation holding the priority of the alert
   --debug                      Enable debug output of the server
   --version                    Show application version.
+  --details=0                  Amount of details in messages. 0 = default, 1 = extended
```

Comment:
I *really* like this idea! I'm contemplating the name of the parameter and wonder if we'd ever find reason for a `--details=2+` level? I'm not sure what additional details we could add to an alert so I'm guessing that the parameter may be better served as just `--extended_details` or something like that.

Could you also add a note about what details show up when this parameter is toggled? Perhaps a simple string like `When enabled, alerts are presented in HTML format and include colorized status (FIR|RES), alert start time, and a link to the generator of the alert.`

Reply:
I thought we might let room for more different visualizations in the future, but I am perfectly fine with a single extended_details flag, so I changed it.